src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -8,7 +8,7 @@
  * A read-only immutable interface for a Person in the addressbook.
  * Implementations should guarantee: details are present and not null, field values are validated.
  */
-public interface ReadOnlyPerson {
+public interface ReadOnlyPerson extends Printable{
```

Comment:
since this interface is implemented by the Person class, the new methods also need to be implemented in that class. Alternative is to have the Person class implementing Printable instead.